AndriiDryha/HW11/PRACTICAL TASKS [LESSON 11].py

```diff
@@ -0,0 +1,52 @@
+# # Task 1
+
+# def number_type():
+#     try:
+#         age = int(input('Enter your age:\n'))
+#         try:
+#             if age > 0 and age % 2 == 0:
+#                 print(f"Age {age} is even.")
+#             elif age > 0 and  age % 2 != 0:
+#                 print(f"Age {age} is odd.")
+#             elif age <= 0:
+#                 raise ValueError()  
+#         except ValueError:
+#             print("Age should be above 0")  
+#             number_type()     
+#     except:
+#         print('Incorrect input.')
+#         number_type()
+       
+# number_type()
+
+# Task 2
+
```

Comment:
good job 
but look at this version of code
def day_number():
    try:
        number = int(input('Enter the number of the day:\n'))
        if 1 <= number <= 7:
            days = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"]
            print(f"{number} is {days[number - 1]}.")
        else:
            print("Number should be ranging between 1 and 7.")
            day_number()
    except ValueError:
        print("Invalid input. Please enter a number between 1 and 7.")
        day_number()
    except KeyboardInterrupt:
        print("\nExiting...")
    except Exception as e:
        print(f"An error occurred: {e}")
        day_number()

day_number()